schemas/context/identity.schema.json

```diff
@@ -20,6 +20,14 @@
           "type": "string",
           "description": "Identity of the consumer in the related namespace."
         },
+        "xdm:additionalIDs": {
```

Comment:
Additional Ids are also needed to support multiple ECID --> CRMID mappings that are provided by co-op and private graph customers. The support for additional Identities should be done outside of the Identity object where each id specified, is of type xdm:Identity. Either in the EndUserIds structure or in a separate Identity extension for ExperiencEvent.

Reply:
@harleensahni  please review with Rahul, I agree on your proposal, this is likely just something you need to talk through.

Reply:
@biswas-adobe  has concerns with Identity itself being modified. It's beginning to look like a better approach to using EndUserID here (as well as in Experience Event) is to go with an array of Identities. 

Modeling with additional IDs is not compatible with #459 since it does not allow for different authentication states for each id in a namespace.

Reply:
@harleensahni I also feel like if would be better to use an array of Identity.  This is how the same problem was handled in Profile.

Reply:
@kstreeter @biswas-adobe I've changed this to be an array of Identity